daemon/stats_collector_unix.go

```diff
@@ -35,6 +36,11 @@ func (daemon *Daemon) newStatsCollector(interval time.Duration) *statsCollector
 		clockTicksPerSecond: uint64(system.GetClockTicks()),
 		bufReader:           bufio.NewReaderSize(nil, 128),
 	}
+	meminfo, err := sysinfo.ReadMemInfo()
```

Comment:
@cpuguy83 `newStatsCollector` only be called once on daemon init in https://github.com/docker/docker/blob/master/daemon/daemon.go#L798 and it store in `daemon.statsCollector.machineMemory`